### ISSUE TITLE
Pinto incorrectly categorizes a module as 'included in perl core' (DB_File)

### ISSUE THREAD

**Issue by gmoon** [thaljef/Pinto#218]
Pinto reports that the module DB_File is included in perl 5.018004 core and declines the pull, but this module is not included in perl core until 5.022.

Having a way to `--force` the pull would suffice, if version-specific core metadata is too cumbersome.

    > source ~/opt/local/pinto/etc/bashrc; ~/opt/local/pinto/bin/pinto -v -r dists pull --recurse DB_File
    Pulling target DB_File~0 to stack master
    Skipping DB_File~0: included in perl 5.018004 core
    Target DB_File~0 is already on stack master
    No changes were made

    > perl -v
    This is perl 5, version 18, subversion 4 (v5.18.4) built for x86_64-linux-thread-multi

    > perl -MDB_File
    Can't locate DB_File.pm in @INC (you may need to install the DB_File module)

**Comment by thaljef** [comment]
>  but this module is not included in perl core until 5.022.

Are you sure about that? Unless Module::CoreList is incorrect, it appears that DB_File has been in core for quite a while (see below). Perhaps someone removed your DB_File.pm. Or perhaps your linux vendor doesn't provide a complete Perl. Or maybe your @INC is broken.

```
$ corelist -a DB_File

Data for 2015-06-01
DB_File was first released with perl 5
  5          undef
  5.001      undef
  5.002      1.01
  5.00307    1.03
  5.004      1.14
  5.00405    1.15
  5.005      1.60
  5.00503    1.65
  5.00504    1.807
  v5.6.0     1.72
  v5.6.1     1.75
  v5.6.2     1.806
  v5.7.3     1.804
  v5.8.0     1.804
  v5.8.1     1.806
  v5.8.2     1.807
  v5.8.3     1.808
  v5.8.4     1.808
  v5.8.5     1.809
  v5.8.6     1.810
  v5.8.7     1.811
  v5.8.8     1.814
  v5.8.9     1.817
  v5.9.0     1.806
  v5.9.1     1.808_01
  v5.9.2     1.811
  v5.9.3     1.814
  v5.9.4     1.814
  v5.9.5     1.815
  v5.10.0    1.816_1
  v5.10.1    1.820
  v5.11.0    1.820
  v5.11.1    1.820
  v5.11.2    1.820
  v5.11.3    1.820
  v5.11.4    1.820
  v5.11.5    1.820
  v5.12.0    1.820
  v5.12.1    1.820
  v5.12.2    1.820
  v5.12.3    1.820
  v5.12.4    1.820
  v5.12.5    1.820
  v5.13.0    1.820
  v5.13.1    1.820
  v5.13.2    1.820
  v5.13.3    1.820
  v5.13.4    1.820
  v5.13.5    1.820
  v5.13.6    1.820
  v5.13.7    1.820
  v5.13.8    1.820
  v5.13.9    1.821
  v5.13.10   1.821
  v5.13.11   1.821
  v5.14.0    1.821
  v5.14.1    1.821
  v5.14.2    1.821
  v5.14.3    1.821
  v5.14.4    1.821
  v5.15.0    1.822
  v5.15.1    1.822
  v5.15.2    1.824
  v5.15.3    1.824
  v5.15.4    1.824
  v5.15.5    1.824
  v5.15.6    1.824
  v5.15.7    1.824
  v5.15.8    1.826
  v5.15.9    1.826
  v5.16.0    1.826
  v5.16.1    1.826
  v5.16.2    1.826
  v5.16.3    1.826
  v5.17.0    1.826
  v5.17.1    1.826
  v5.17.2    1.826
  v5.17.3    1.826
  v5.17.4    1.827
  v5.17.5    1.827
  v5.17.6    1.827
  v5.17.7    1.827
  v5.17.8    1.827
  v5.17.9    1.827
  v5.17.10   1.827
  v5.17.11   1.827
  v5.18.0    1.827
  v5.18.1    1.827
  v5.18.2    1.827
  v5.18.3    1.827
  v5.18.4    1.827
  v5.19.0    1.827
  v5.19.1    1.828
  v5.19.2    1.829
  v5.19.3    1.829
  v5.19.4    1.829
  v5.19.5    1.829
  v5.19.6    1.831
  v5.19.7    1.831
  v5.19.8    1.831
  v5.19.9    1.831
  v5.19.10   1.831
  v5.19.11   1.831
  v5.20.0    1.831
  v5.20.1    1.831
  v5.20.2    1.831
  v5.21.0    1.831
  v5.21.1    1.831
  v5.21.2    1.831
  v5.21.3    1.831
  v5.21.4    1.831
  v5.21.5    1.831
  v5.21.6    1.831
  v5.21.7    1.834
  v5.21.8    1.835
  v5.21.9    1.835
  v5.21.10   1.835
  v5.21.11   1.835
  v5.22.0    1.835
```

**Comment by gmoon** [comment]
I stand corrected.  I built perl from source and didn't have Berkley DB installed, so DB_File was not installed.  Thanks for the pointer on Module::CoreList and corelist.
